code/game/machinery/computer/communications.dm

```diff
@@ -500,10 +500,13 @@
 	if(ticker.mode.name == "meteor")
 		return
 
-	shuttle_master.cancelEvac(user)
-	log_game("[key_name(user)] has recalled the shuttle.")
-	message_admins("[key_name_admin(user)] has recalled the shuttle - [formatJumpTo(user)].", 1)
-	return
+	if(shuttle_master.cancelEvac(user))
+		log_game("[key_name(user)] has recalled the shuttle.")
+		message_admins("[key_name_admin(user)] has recalled the shuttle - [formatJumpTo(user)].", 1)
+	else
+		to_chat(user, "The shuttle can not be recalled! Central Orders!")
+		log_game("[key_name(user)] has tried and failed to recall the shuttle.")
+		message_admins("[key_name_admin(user)] has tried and failed to recall the shuttle - [formatJumpTo(user)].", 1)

```

Comment:
I realize this formatJumpTo is consistent with the rest of shuttle code, but I don't know why. It should use the `ADMIN_JMP(user)` define, or better yet, `ADMIN_FLW(user)`.